mods/seasonal/moves.js

```diff
@@ -191,7 +191,6 @@ let BattleMovedex = {
 				this.add('-message', `${source.name} made a wish!`);
 			}
 			for (const ally of side.pokemon) {
-				if (ally.hasAbility('soundproof')) continue;
```

Comment:
can you also remove this from víbora? i dont think that needs a soundproof check

Reply:
This wasn't in Vibora